etc/kernelspecs/python_docker/kernel.json

```diff
@@ -6,7 +6,8 @@
       "class_name": "enterprise_gateway.services.processproxies.docker_swarm.DockerSwarmProcessProxy",
       "config": {
         "image_name": "elyra/kernel-py:VERSION"
-      }
+      },
+    "debugger": true
```

Comment:
This is at the wrong level and still shows as a (mal-indented) attribute of `process_proxy` when it should be an attribute of `metadata`. 

Same goes for all of the `kernel.json` updates.